index.html

```diff
@@ -5045,9 +5045,14 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<rdef>meter</rdef>
 			<div class="role-description">
 				<p>An <a>element</a> that represents a scalar measurement within a known range, or a fractional value. See related <rref>progressbar</rref>.</p>
-				<p>Authors MUST NOT use the <code>meter</code> role to represent a value whose minimum and maximum values are unknown. In addition, authors MUST use <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to make the minimum and maximum values available to assistive technologies.</p>
+				<p>Authors MAY set <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to make the minimum and maximum values available to assistive technologies. Otherwise, their implicit values follow the same rules as <code>&lt;input[type="range"]&gt;</code> in [[HTML]]:</p>
```

Comment:
Min and max will be available to assistive technologies whether they're author-set or implied...

```suggestion
				<p>Authors MAY set <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to indicate the minimum and maximum values for the <code>meter</code>. Otherwise, their implicit values follow the same rules as <code>&lt;input[type="range"]&gt;</code> in [[HTML]]:</p>
```